diff --git a/node_modules/llama.rn/android/build.gradle b/node_modules/llama.rn/android/build.gradle
index 10d8fd3..8c997df 100644
--- a/node_modules/llama.rn/android/build.gradle
+++ b/node_modules/llama.rn/android/build.gradle
@@ -127,6 +127,7 @@ def REACT_NATIVE_VERSION = reactProperties.getProperty("VERSION_NAME")
 def REACT_NATIVE_MINOR_VERSION = REACT_NATIVE_VERSION.startsWith("0.0.0-") ? 1000 : REACT_NATIVE_VERSION.split("\\.")[1].toInteger()
 
 android {
+  namespace "com.rnllama"
   ndkVersion getExtOrDefault("ndkVersion")
   def ndkVersionMajor = ndkVersion.split("\\.")[0].toInteger()
   if (ndkVersionMajor < 24) {
diff --git a/node_modules/llama.rn/android/src/main/AndroidManifest.xml b/node_modules/llama.rn/android/src/main/AndroidManifest.xml
index 4de38a7..0a0938a 100644
--- a/node_modules/llama.rn/android/src/main/AndroidManifest.xml
+++ b/node_modules/llama.rn/android/src/main/AndroidManifest.xml
@@ -1,4 +1,3 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.rnllama">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
 </manifest>
